docs/tutorial/image_seg.md

```diff
@@ -1,3 +1,168 @@
+
+Generate your model configuration file interactively by running `blueoil.sh init` command.
+
+    $ ./blueoil.sh init
+
+This is an example of initialization.
+
+```
+#### Generate config ####
+your model name ():  camvid
+choose task type  semantic_segmentation
+choose network  LmSegnetV1Quantize
+choose dataset format  CamvidCustom
+training dataset path:  {dataset_dir}
+set validataion dataset? (if answer no, the dataset will be separated for training and validation by 9:1 ratio.)  yes
+test dataset path:  {dataset_dir}
+batch size (integer):  8
+image size (integer x integer):  360x480
+how many epochs do you run training (integer):  1000
+choose optimizer: Adam
+initial learning rate:  0.001
+choose learning rate schedule ({epochs} is the number of training epochs you entered before):  '3-step-decay' -> learning rate decrease by 1/10 on {epochs}/3 and {epochs}*2/3 and {epochs}-1
+enable data augmentation?  Yes
+Please choose augmentors:  done (5 selections)
+-> select Brightness, Color, Contrast, FlipLeftRight, Hue
+apply quantization at the first layer?  no
+```
+
+## Train a network model
+
+Train your model by running `blueoil.sh train` command with model configuration.
+
+    $ ./blueoil.sh train config/{Model name}.yml
+
+When training is started, training log and checkpoints are generated under `./saved/{Mode name}_{TIMESTAMP}` directory.
+
+Training is running on TensorFlow backend. So you can use TensorBoard to visualize your training process. 
+
+    $ ./blueoil.sh tensorboard saved/{Model name}_{TIMESTAMP} {Port}
+
+- Learning Rate / Loss
+<img src="../_static/semantic_segmentation_loss.png">
+
+- Metrics / IOU
+<img src="../_static/semantic_segmentation_iou.png">
+
+- Images / Overlap Output Input
+<img src="../_static/semantic_segmentation_overwrap.png">
+
+## Convert training result to FPGA ready format.
+
+Convert trained model to executable binary files for x86, ARM, and FPGA.
+Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
+
+    $ ./blueoil.sh convert config/{Model name}.yml saved/{Mode name}_{TIMESTAMP}
+
+`blueoil.sh convert` automatically executes some conversion processes.
+- Convert Tensorflow checkpoint to protocol buffer graph.
+- Optimize graph
+- Generate source code for executable binary
+- Compile for x86, ARM and FPGA
+
+If conversion is successful, output files are generated under `./saved/{Mode name}_{TIMESTAMP}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.
+
+```
+output
+ ├── fpga (include preloader and FPGA configuration file)
+ │   ├── preloader-mkpimage.bin
+ │   └── soc_system.rbf
+ ├── models
+ │   ├── lib (include trained model library)
+ │   │   ├── lib_arm.so
+ │   │   ├── lib_fpga.so
+ │   │   └── lib_x86.so
+ │   └── meta.yaml (model configuration)
+ ├── python
+ │   ├── lmnet (include pre-process/post-process)
+ │   ├── README.md
+ │   ├── requirements.txt
+ │   ├── run.py (inference script)
+ │   └── usb_camera_demo.py (demo script for object detection)
```

Comment:
`usb_camera_demo.py (demo script for object detection)` -> `usb_camera_demo.py (demo script for object detection and classification)` looks better following other tutorials.

Reply:
Fixed.